### PR TITLE
Fix Execution of Scripts

### PR DESCRIPTION
**Pull request by lharzenetter**
#### Short Description
Currently, if a Node Type inherits from another, the Management Bus does not travers the inheritance hierarchy to find interfaces and operations.

#### Proposed Changes

  * Use the new method that identifies if a specific interface is defined in the hierarchy of a Node Type and returns the corresponding set of management operations.

### CODE REVIEW

File: .idea/runConfigurations/Container.xml [hunk 1]
```diff
@@ -19,7 +19,7 @@
     <predefined_log_file id="Tomcat Host Manager" />
     <predefined_log_file id="Tomcat Localhost Access" />
     <RunnerSettings RunnerId="Debug">
-      <option name="DEBUG_PORT" value="63450" />
+      <option name="DEBUG_PORT" value="61544" />
```
Comment (nyuuyn) [inline .idea/runConfigurations/Container.xml:22]: Is this change necessary ?

File: org.opentosca.bus/org.opentosca.bus.management/src/main/java/org/opentosca/bus/management/utils/MBUtils.java [hunk 2]
```diff
@@ -141,19 +141,21 @@ public static NodeTemplateInstance getAbstractOSReplacementInstance(NodeTemplate
...
-        boolean isOs = exposedInterfaces.stream()
-            .filter(tInterface -> Interfaces.OPENTOSCA_DECLARATIVE_INTERFACE_OPERATINGSYSTEM.equals(tInterface.getName()))
-            .anyMatch(os -> doesInterfaceContainOperation(os, Interfaces.OPENTOSCA_DECLARATIVE_INTERFACE_OPERATINGSYSTEM_RUNSCRIPT)
-                && doesInterfaceContainOperation(os, Interfaces.OPENTOSCA_DECLARATIVE_INTERFACE_OPERATINGSYSTEM_TRANSFERFILE));
-        boolean isDocker = exposedInterfaces.stream()
-            .filter(tInterface -> Interfaces.OPENTOSCA_DECLARATIVE_INTERFACE_DOCKERCONTAINER.equals(tInterface.getName()))
-            .anyMatch(os -> doesInterfaceContainOperation(os, Interfaces.OPENTOSCA_DECLARATIVE_INTERFACE_DOCKERCONTAINER_RUNSCRIPT)
-                && doesInterfaceContainOperation(os, Interfaces.OPENTOSCA_DECLARATIVE_INTERFACE_DOCKERCONTAINER_TRANSFERFILE));
+
+        TInterface interfaceOfNodeType = ModelUtils.getInterfaceOfNodeType(csar, nodeType, Interfaces.OPENTOSCA_DECLARATIVE_INTERFACE_OPERATINGSYSTEM);
```
Comment (nyuuyn) [inline org.opentosca.bus/org.opentosca.bus.management/src/main/java/org/opentosca/bus/management/utils/MBUtils.java:149]: wouldn't it be more clear code if we used a method like getAllInterfacesOfNodeType() or getInheritedInterfacesOfNodeType() ?

**Review by nyuuyn — APPROVED** [review]
lgtm

**Review by zimmerml — APPROVED** [review]

**Review by wederbn — APPROVED** [review]
LGTM
